R/current_compile_log.R

```diff
@@ -0,0 +1,12 @@
+current_compile_log <- function(log_file_path = getwd(), base_name) {
+  no_log_file <- sum(file.exists(log_file_path, hidden.files = TRUE)) == 0
+  if (no_log_file) {
+    initialize_log <- list(initialize = TRUE, timestamp = Sys.time())
```

Comment:
Minor, but best ensure `timestamp` will have the same format when initialised here, and for further logs.

Reply:
:+1: